AudioManagerDaemon/src/main.cpp

```diff
@@ -384,6 +384,14 @@ iControlSender.setControllerReady();
  */
 int main(int argc, char *argv[], char** envp)
 {
+    struct sched_param param;
+    param.sched_priority = 50;//mid rt proprity
+    if (sched_setscheduler(0, SCHED_FIFO, & param) != 0) 
```

Comment:
This is not really a good implementation. We have to keep this as an configuration parameter for the daemon.
Both aspects, priority and policy, should be configurable by e.g. configuration parameter.